### ISSUE TITLE
MBO Support BPMN2 send and receive

### ISSUE THREAD

**Issue by adamfowleruk** [adamfowleruk/scxmltocpf#10]
Once event handling implemented, and sub processes, should be able to easily implement sending a message (fire and forget) and receiving a message (async messaging).

**Comment by adamfowleruk** [comment]
Depends upon #4

**Comment by adamfowleruk** [comment]
Depends upon event handling too - postponed until sprint 004
